File: app/src/main/java/com/examples/android/androidtrainingbzu/MainActivity.java
package com.examples.android.androidtrainingbzu;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startNewActivity(View view) {
        switch (view.getId()){
            case R.id.btn_startLongActivity:
                startActivity(new Intent(this,LongTextActivity.class));
                break;
            case R.id.btn_startLifecycleActivity:
                startActivity(new Intent(this,ActivityLifcycle.class));
                break;
            case R.id.btn_startActivityResul:
                startActivity(new Intent(this,ForResultsActivity.class));
                break;
            case R.id.btn_startimplicitActivity:
                startActivity(new Intent(this,ImplicitActivity.class));
                break;
            case R.id.btn_startView1Activity:
                startActivity(new Intent(this,ViewsActivity_1.class));
                break;
            case R.id.btn_startWebviewActivity:
                startActivity(new Intent(this,WebviewActivity.class));
                break;
            case R.id.btn_startDialogsActivity:
                startActivity(new Intent(this,DialogsActivity.class));
                break;
            case R.id.btn_startScaleGestureDetectorActivity:
                startActivity(new Intent(this,ScaleGestureDetectorActivity.class));
                break;
            case R.id.btn_startMotionEventActivity:
                startActivity(new Intent(this,MotionEventActivity.class));
                break;
            case R.id.btn_startMenuActivity:
                startActivity(new Intent(this,MenusActivity.class));
                break;
            case R.id.btn_startTabsActivity:
                startActivity(new Intent(this,TabsActivity.class));
                break;
            case R.id.btn_startLoadImagesActivity:
                startActivity(new Intent(this,LoadImagesAsyncAndLoaderActivity.class));
                break;
            case R.id.btn_startChecFormkActivity:
                startActivity(new Intent(this,FormCheckActivity.class));
                break;
            case R.id.btn_startCustomBrodcastActivity:
                startActivity(new Intent(this,BroadcastActivity.class));
                break;
            case R.id.btn_startCustomDownloadManagerActivity:
                startActivity(new Intent(this,CustomDownloadManagerActivity.class));
                break;
            case R.id.btn_downloadFileUsingService:
                startActivity(new Intent(this,StartIntentServiceActivity.class));
                break;
            case R.id.btn_JobServiceActivity:
                startActivity(new Intent(this,JobServiceActivity.class));
                break;
            case R.id.btn_openNotification:
                startActivity(new Intent(this,StartNotificationActivity.class));
                break;
            case R.id.btn_AlarmActivity:
                startActivity(new Intent(this,AlarmActivity.class));
                break;
            case R.id.btn_languageAndSharedPreferencesActivity:
                startActivity(new Intent(this,LanguageAndSharedPreferencesActivity.class));
                break;
            case R.id.btn_startRecyclerViewActivity:
                startActivity(new Intent(this,HR_MainActivity.class));
                break;
        }
    }
}

File: app/src/main/java/com/examples/android/androidtrainingbzu/MainActivity.kt
package com.examples.android.androidtrainingbzu

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.examples.android.androidtrainingbzu.ActivityLifcycle
import com.examples.android.androidtrainingbzu.AlarmActivity
import com.examples.android.androidtrainingbzu.BroadcastActivity
import com.examples.android.androidtrainingbzu.ForResultsActivity
import com.examples.android.androidtrainingbzu.FormCheckActivity
import com.examples.android.androidtrainingbzu.ImplicitActivity
import com.examples.android.androidtrainingbzu.JobServiceActivity
import com.examples.android.androidtrainingbzu.LanguageAndSharedPreferencesActivity
import com.examples.android.androidtrainingbzu.StartNotificationActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun startNewActivity(view: View) {
        when (view.id) {
            R.id.btn_startLongActivity -> startActivity(Intent(this, LongTextActivity::class.java))
            R.id.btn_startLifecycleActivity -> startActivity(Intent(this,
                ActivityLifcycle::class.java))
            R.id.btn_startActivityResul -> startActivity(Intent(this,
                ForResultsActivity::class.java))
            R.id.btn_startimplicitActivity -> startActivity(Intent(this,
                ImplicitActivity::class.java))
            R.id.btn_startView1Activity -> startActivity(Intent(this, ViewsActivity_1::class.java))
            R.id.btn_startWebviewActivity -> startActivity(Intent(this,
                WebviewActivity::class.java))
            R.id.btn_startDialogsActivity -> startActivity(Intent(this,
                DialogsActivity::class.java))
            R.id.btn_startScaleGestureDetectorActivity -> startActivity(Intent(this,
                ScaleGestureDetectorActivity::class.java))
            R.id.btn_startMotionEventActivity -> startActivity(Intent(this,
                MotionEventActivity::class.java))
            R.id.btn_startMenuActivity -> startActivity(Intent(this, MenusActivity::class.java))
            R.id.btn_startTabsActivity -> startActivity(Intent(this, TabsActivity::class.java))
            R.id.btn_startLoadImagesActivity -> startActivity(Intent(this,
                LoadImagesAsyncAndLoaderActivity::class.java))
            R.id.btn_startChecFormkActivity -> startActivity(Intent(this,
                FormCheckActivity::class.java))
            R.id.btn_startCustomBrodcastActivity -> startActivity(Intent(this,
                BroadcastActivity::class.java))
            R.id.btn_startCustomDownloadManagerActivity -> startActivity(Intent(this,
                CustomDownloadManagerActivity::class.java))
            R.id.btn_downloadFileUsingService -> startActivity(Intent(this,
                StartIntentServiceActivity::class.java))
            R.id.btn_JobServiceActivity -> startActivity(Intent(this,
                JobServiceActivity::class.java))
            R.id.btn_openNotification -> startActivity(Intent(this,
                StartNotificationActivity::class.java))
            R.id.btn_AlarmActivity -> startActivity(Intent(this, AlarmActivity::class.java))
            R.id.btn_languageAndSharedPreferencesActivity -> startActivity(Intent(this,
                LanguageAndSharedPreferencesActivity::class.java))
            R.id.btn_startRecyclerViewActivity -> startActivity(Intent(this,
                HR_MainActivity::class.java))
        }
    }
}